.circleci/config.yml

```diff
@@ -8,40 +8,12 @@ version: 2.1
 jobs:
   lint-vet-fmt:
     docker:
-      - image: cimg/go:1.22
+      # If you adjust this version, also adjust the GOLANGCI_LINT_VERSION in the
+      # Makefile
+      - image: golangci/golangci-lint:v1.60.1-alpine
     steps:
-      # TODO(AUT-158): this linting always succeeds even if there are errors, and future correct linting may need correct cgo compliation
       - checkout
-      - run:
-          name: check crypto11 not used in signers
-          command: |
-            make check-no-crypto11-in-signers
-            make show-lints
-      - run:
-          name: run golint
-          command: |
-            make install-golint lint
-            make show-lints
-            make -C tools/autograph-monitor lint
-            make -C verifier/contentsignature lint
-      - run:
-          name: run gofmt
-          command: |
-            make -s fmt-diff | tee fmt.diff
-            test -z "$(cat fmt.diff)"
-      - run:
-          name: run go vet
-          command: |
-            make vet
-            make -C tools/autograph-monitor vet
-            make -C verifier/contentsignature vet
-      - run:
-          name: run staticcheck
-          command: |
-            make install-staticcheck staticcheck
-            make show-lints
-            make -C tools/autograph-monitor staticcheck
-            make -C verifier/contentsignature staticcheck
+      - run: golangci-lint run
```

Comment:
didn't you already add this in #951 ?

Does the base branch need to be rebased?

Reply:
I forgot to edit this PR's base branch! Fixed

Reply:
Thank you! Resolved